src/AggregateFunctions/ReservoirSamplerDeterministic.h

```diff
@@ -190,7 +190,11 @@ class ReservoirSamplerDeterministic
             /// TODO: After implementation of "versioning aggregate function state",
             /// change the serialization format.
             Element elem;
+#pragma clang diagnostic push
```

Comment:
I love how we keep getting more and more warnings that this code was awful to begin with 🥲